JsonSchema/RegexFormat.cs

```diff
@@ -16,7 +15,7 @@ public class RegexFormat : Format
 	/// </summary>
 	/// <param name="key">The format key.</param>
 	/// <param name="regex">The regular expression.</param>
-	public RegexFormat(string key, [RegexPattern] string regex)
+	public RegexFormat(string key, string regex)
```

Comment:
Removing this removes syntax highlighting for regex strings for consumers.  I find this very useful, and I expect other R# uses, too.

Reply:
This is an IDE XP thing, that may be only useful for devs using the R# in their environment, 

As of NET 7 moving forward we can use: `[StringSyntax(StringSyntaxAttribute.Regex)]`

As you can see in this blog post:

https://devblogs.microsoft.com/dotnet/regular-expression-improvements-in-dotnet-7

Reply:
Is that available as part of a .net standard 2.0 package? .net 7 features aren't available to these libraries.

Reply:
> I'm not sure why you're removing the JetBrains package from the main pkg but adding it to the extensions.

To reduce the dependency graph, you will eventually only use this package if really need the `DataGeneration` library.

Reply:
> Is that available as part of a .net standard 2.0 package? .net 7 features aren't available to these libraries.

Yes, its only for .Net 7+ libraries, don't apply here. The fewer dependencies the better + helps also increase adoption.

Reply:
I'm confused.  You're recommending that I replace the Jetbrains attribute with something that's only available in .net 7+, but these are .net standard 2.0 libs.

I'm happy to consider this as part of #390, but for now I'd prefer to stick with it as it is.

Reply:
@gregsdennis, no, I did suggest it but didn't realized this attribute was not available to .Net Standard 2.0.

So, my suggestion would be to remove this dependency from the main package as a standard way of reducing the surface and dependencies that are not strictly necessary the project to work.

Reply:
It's a developer hint and convenience (dev UX is important to me).  I've had this conversation multiple times, and I'd like to have it.  If it's actually causing problems for you, I'm happy to talk about that, but I'm not going to remove it just at the mere suggestion.

In the end, they're non-functional, and the Jetbrains library is tiny (75 KB), so the impact to anything runtime is negligible.

Reply:
I wont bring this further, the more dependencies you add to a package less likely you will see it being used across multiple places. Companies normally have compliance rules that restrict unecessary packages and this just adds more friction, but that's my view, my understanding. 

I'm dropping this now.